.travis.yml

```diff
@@ -33,6 +33,7 @@ script:
   - stty cols 200
   ## need to fix above docker-engine install to not require sudo
   - sudo lc ci
+  - ./blackbox-test/build-examples.sh
```

Comment:
you need to call this with `sudo`, the way docker gets installed in the travis build requires `sudo` to run any docker commands (we could probably fix that, but doesn't seem to be a huge deal).